src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -94,6 +94,9 @@ internal JaegerExporter(JaegerExporterOptions options, TProtocolFactory protocol
         /// <inheritdoc/>
         public override ExportResult Export(in Batch<Activity> activityBatch)
         {
+            // Prevent Jaeger's HTTP operations from being instrumented.
+            using var scope = SuppressInstrumentationScope.Begin();
```

Comment:
If you do this here, we probably don't also need it here:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/72ade321767166ad1cb15786db9240c6416eb9ba/src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerHttpClient.cs#L68

Where it is on the JaegerHttpClient is slightly more efficient for the thrift/socket-based client. But I'm not against having it here just in case someone has socket instrumentation.

Reply:
Yea I missed the one in JaegerHttpClient. I'll revert this for now to avoid the (small) perf hit while using thrift. If we hear about socket instrumentation, we can move it here.